src/unix/notbsd/linux/musl/b32/arm.rs

```diff
@@ -332,7 +332,5 @@ pub const TIOCCONS: ::c_int = 0x541D;
 pub const SYS_gettid: ::c_long = 224;
 pub const SYS_perf_event_open: ::c_long = 364;
 
-pub const POSIX_MADV_DONTNEED: ::c_int = 0;
-pub const RUSAGE_CHILDREN: ::c_int = 1;
```

Comment:
Backwards compatibility requires that we don't delete these definitions, so perhaps the values could just get corrected with the updated ones?